src/Containers/HorizontalBarChart/HorizontalBarChart.tsx

```diff
@@ -0,0 +1,168 @@
+import React,{useCallback, useLayoutEffect, useRef,useState,useMemo} from 'react'
+import {BarChart,XAxis,YAxis,Bar,Text,Cell} from 'recharts'
+import styled from 'styled-components'
+import Theme from '@Themes/ThemeTemplate'
+import {MainTheme} from '@Themes'
+import {Mixins} from '@Utils'
+import {Paragraph} from '@Text'
+
+const FLEX_EQUAL_SPACE=1
+const SUMMARY_BACKGROUND_COLOR='#d3d3d3'
+const BAR_LABEL_LEFT_MARGIN=5
+const SUMMARY_CONTAINER_MARGIN=14
+const IS_TOTAL_COLOR='#d3d3d3'
+const GREEN_COLOR_MARGIN=5
+const ORANGE_COLOR_MARGIN=15
+const BAR_SIZE=20
+
+interface IDataItem{
+    label:string;
+    value:number;
+    isTotal:boolean;
+}
+
+interface IChartOptionalProperties{
```

Comment:
Why not do IChartProperties for ...props or even ...ChartProperties so that you can edit anything